test-support/reference-trie/src/substrate.rs

```diff
@@ -0,0 +1,725 @@
+// Copyright (C) Parity Technologies (UK) Ltd.
```

Comment:
Should remove test-support/reference-trie/src/substrate-like.rs (can be in a second time, might be some slight difference??).

Reply:
Yeah, I think we can remove that one if we're going to have the real thing. I just didn't want to needlessly bloat up this PR with more changes.